lib/player_selector.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class PlayerSelector
+  PLAYER_OPTIONS = {
+    '1' => ['Player 1', 'Player 2'],
+    '2' => ['Computer', 'Player 1']
+  }.freeze
```

Comment:
Instead of initializing names here, what do you think about initializing `Player` objects with their names and markers? In the future when we define custom markers, we could pass in player objects here. Something to think about